source/appModules/foobar2000.py

```diff
@@ -77,25 +84,27 @@ def script_reportRemainingTime(self,gesture):
 			# Translators: Reported if the remaining time can not be calculated in Foobar2000
 			msg = _("Unable to determine remaining time")
 		else:
-			parsedElapsedTime = parseIntervalToTimestamp(elapsedTime)
 			parsedTotalTime = parseIntervalToTimestamp(totalTime)
-			remainingTime = parsedTotalTime - parsedElapsedTime
-			msg = time.strftime(getOutputFormat(remainingTime), time.gmtime(remainingTime))
+			parsedElapsedTime = parseIntervalToTimestamp(elapsedTime)
+			msg = getRemainingTime(parsedElapsedTime, parsedTotalTime) + " remaining"
 		ui.message(msg)
 	# Translators: The description of an NVDA command for reading the remaining time of the currently playing track in Foobar 2000.
 	script_reportRemainingTime.__doc__ = _("Reports the remaining time of the currently playing track, if any")
 
 	def script_reportElapsedTime(self,gesture):
 		elapsedTime = self.getElapsedAndTotalIfPlaying()[0]
 		if elapsedTime is not None:
-			ui.message(elapsedTime)
+			ui.message(elapsedTime + " elapsed")

```

Comment:
This needs to be a translation string, otherwise elapsed won't be translated

```suggestion
			# Translators: The time elapsed time of a currently playing track in Foobar 2000.
			# %s is replaced with the time elapsed.
			ui.message(_("%s elapsed" % elapsedTime)
```